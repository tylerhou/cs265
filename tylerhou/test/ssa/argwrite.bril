@main {
  a: int = const 10;
  cond: bool = const true;
  br cond .here .there;
.here:
  a: int = const 5;
.there:
  print a;
}
