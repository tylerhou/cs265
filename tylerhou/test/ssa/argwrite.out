@main {
.__b1:
  a0: int = const 10;
  cond: bool = const true;
  br cond .here .there(a0);
.here:
  a1: int = const 5;
  jmp .there(a1);
.there(a2: int):
  print a2;
  ret;
}
